source/Cute.Lib/Contentful/BulkActions/Actions/BulkActionBase.cs

```diff
@@ -312,7 +311,10 @@ protected async Task ChangePublishRequiredEntries(List<BulkItem> items, BulkActi
 
     private async Task<BulkActionResponse> SendBulkChangePublishRequestViaHttp(BulkAction bulkAction, IEnumerable<BulkItem> items, Action<BulkActionProgressEvent>? progressUpdater)
     {
-        var bulkEndpoint = new Uri($"https://api.contentful.com/spaces/{_contentfulConnection.Options.SpaceId}/environments/{_contentfulConnection.Options.Environment}/bulk_actions/{bulkAction.ToString().ToLower()}");
+        var spaceId = (await _contentfulConnection.GetDefaultSpaceAsync()).SystemProperties.Id;
```

Comment:
would it be consistent with the intended idea to move this inside of contentfulConnection?